src/payment/checkout/payment-form/flex-microform/FlexMicroformField.jsx

```diff
@@ -29,9 +30,16 @@ class FlexMicroformField extends React.Component {
   }
 
   initialize() {
-    if (!window.microform || this.field !== null || this.props.microformStatus !== STATUS_READY) {
+    if (!window.microform || this.props.microformStatus !== STATUS_READY) {
```

Comment:
I think you might be able to make this clearer by just passing `microform` as a `prop`, rather than using `window.microform`? Then you could just check if the `microform` prop went from null to non-null, and re-initialize?

Reply:
Although, maybe that would require storing the `microform` in `state`, and maybe that's not kosher? I think it's probably worth a try, though. @mulby, why did you stash `microform` on the `window` in the first place?

Reply:
yeah, I had it in state initially, but that is an anti-pattern. I needed to stash it on window so that it could be retrieved during the saga execution when the click the submit button here:

https://github.com/edx/frontend-app-payment/blob/2c4d079907a0b077400871e59bda12875ecea888/src/payment/payment-methods/cybersource/service.js#L69

I explicitly changed it to a status prop and window var to avoid passing complex objects down the tree (per earlier feedback from FED experts).

Reply:
Ah, ok, great. Carry on, then.